src/buildercore/terraform.py

```diff
@@ -564,6 +569,230 @@ def render_bigquery(context, template):
+        'name': 'project-with-eks--%s--master' % context['instance_id'],
+        'description': 'Cluster communication with worker nodes',
+        'vpc_id': context['aws']['vpc-id'],
+        'egress': {
+            'from_port': 0,
+            'to_port': 0,
+            'protocol': '-1',
+            'cidr_blocks': ['0.0.0.0/0'],
+        },
+        'tags': aws.generic_tags(context),
+    })
+
+def _render_eks_workers(context, template):
+    template.populate_resource('aws_security_group_rule', 'worker_to_master', block={
+        'description': 'Allow pods to communicate with the cluster API Server',
+        'from_port': 443,
+        'protocol': 'tcp',
+        'security_group_id': '${aws_security_group.master.id}',
+        'source_security_group_id': '${aws_security_group.worker.id}',
+        'to_port': 443,
+        'type': 'ingress',
+    })
+
+    security_group_tags = aws.generic_tags(context)
+    security_group_tags['kubernetes.io/cluster/%s'] = 'owned'
+    template.populate_resource('aws_security_group', 'worker', block={
+        'name': 'project-with-eks--%s--worker' % context['instance_id'],
+        'description': 'Security group for all worker nodes in the cluster',
+        'vpc_id': context['aws']['vpc-id'],
+        'egress': {
+            'from_port': 0,
+            'to_port': 0,
+            'protocol': '-1',
+            'cidr_blocks': ['0.0.0.0/0'],
+        },
+        'tags': security_group_tags,
+    })
+
+    template.populate_resource('aws_security_group_rule', 'worker_to_worker', block={
+        'description': 'Allow worker nodes to communicate with each other',
+        'from_port': 0,
+        'protocol': '-1',
+        'security_group_id': '${aws_security_group.worker.id}',
+        'source_security_group_id': '${aws_security_group.worker.id}',
+        'to_port': 65535,
+        'type': 'ingress',
+    })
+
+    template.populate_resource('aws_security_group_rule', 'master_to_worker', block={
+        'description': 'Allow worker Kubelets and pods to receive communication from the cluster control plane',
+        'from_port': 1025,
+        'protocol': 'tcp',
+        'security_group_id': '${aws_security_group.worker.id}',
+        'source_security_group_id': '${aws_security_group.master.id}',
+        'to_port': 65535,
+        'type': 'ingress',
+    })
+
+    template.populate_resource('aws_security_group_rule', 'eks_public_to_worker', block={
+        'description': "Allow worker to expose NodePort services",
+        'from_port': 30000,
+        'protocol': 'tcp',
+        'security_group_id': '${aws_security_group.worker.id}',
+        'to_port': 32767,
+        'type': 'ingress',
+        'cidr_blocks': ["0.0.0.0/0"],
+    })
+
+    template.populate_resource('aws_iam_role', 'worker', block={
+        'name': '%s--AmazonEKSWorkerRole' % context['stackname'],
+        'assume_role_policy': json.dumps({
+            "Version": "2012-10-17",
```

Comment:
Where is that `Version` coming from?

Reply:
It's the current [API version](https://docs.aws.amazon.com/IAM/latest/UserGuide/reference_policies_elements_version.html)